spec/integration/publish_intent_crud_spec.rb

```diff
@@ -23,14 +23,11 @@
         expect(intent.publish_time).to match_datetime(publish_time)
       end
 
-      it "responds with a created status, and the intent as json" do
+      it "responds with a created status and an empty JSON document" do
         put_json "/publish-intent/vat-rates", data
 
         expect(response.status).to eq(201)
-
-        data = JSON.parse(response.body)
-        expect(data['base_path']).to eq('/vat-rates')
-        expect(data['publish_time']).to match_datetime(publish_time)
+        expect(response.body).to eq('{}')
```

Comment:
That's a good point. I did this to be consistent with the [proposed API guidelines](https://gov-uk.atlassian.net/wiki/display/PUB/RFC+6%3A+API+guidelines):
> Success AND error response bodies MUST be JSON, except where another format is desired (eg CSV)

What I haven't done in this PR is to follow this guideline as it felt like an unrelated change:
> Success AND error responses MUST include a top-level "message" key containing a user-friendly summary of the outcome - eg "Successfully created", "Validation failed",